Tools/TestWebKitAPI/Tests/mac/ContextMenuTests.mm

```diff
@@ -767,6 +767,27 @@ static void swizzledOrderFrontColorPanel(id, SEL, id)
     Util::run(&gotProposedMenu);
 }
 
+TEST(ContextMenuTests, HitTestResultLinkLocalDataSuggestedFilename)
+{
+    auto delegate = adoptNS([[TestUIDelegate alloc] init]);
+
+    __block bool gotProposedMenu = false;
+    [delegate setGetContextMenuFromProposedMenu:^(NSMenu *menu, _WKContextMenuElementInfo *elementInfo, id<NSSecureCoding>, void (^completion)(NSMenu *)) {
+        EXPECT_NOT_NULL(elementInfo.hitTestResult);
+        EXPECT_TRUE([elementInfo.hitTestResult.linkLocalDataSuggestedFilename isEqualToString:@"sunset-in-cupertino-600px.jpg"]);
```

Comment:
EXPECT_WK_STREQ can be used to compare NSStrings to c-strings.  That way if there is a difference it will print out the strings instead of just "false"

Reply:
done